recipes/leftbar/group_row/group_row_variants.story.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="d-stack16 d-w464">
```

Comment:
Super minor nit, but why not use our new stack component and then use gap to space it out instead of padding?

Reply:
In fact, we should start using `<dt-stack>` as much as possible to stress-test it out of Beta.

Reply:
good call, updated variant stories in the leftbar to use `dt-stack` https://github.com/dialpad/dialtone-vue/pull/778/commits/4e50a0857ab80e3bd3720e3c1503285e185976e3